imm_ptl_core/src/main/java/qouteall/imm_ptl/core/portal/custom_portal_gen/CustomPortalGeneration.java

```diff
@@ -49,7 +49,7 @@ public class CustomPortalGeneration {
     );
     
     public static ResourceKey<Registry<CustomPortalGeneration>> registryRegistryKey =
-        ResourceKey.createRegistryKey(new ResourceLocation("imm_ptl:custom_portal_generation"));
+        ResourceKey.createRegistryKey(new ResourceLocation("custom_portal_generation")); // Custom registry namespaces not enabled in forge
```

Comment:
Can you please change the comment to
`// FORGE: Custom registry namespaces not enabled in forge`. Makes it easier to search for it when updating.